src/vaults/SyncRequests.sol

```diff
@@ -94,6 +105,9 @@ contract SyncRequests is BaseInvestmentManager, ISyncRequests {
         address token = vault_.share();
         vault[poolId][scId][assetId] = vaultAddr;
 
+        (, uint256 tokenId) = poolManager.idToAsset(assetId);
+        maxReserve[poolId][scId][asset_][tokenId] = type(uint128).max;
```

Comment:
Business decision we need to make: do we automatically set reserve to max, or force pool admins to initialize it?

Reply:
I lean to automatically setting to max, to not add even more pool setup calls.

Reply:
IMO that's absolutely feasible to have "opt-out < max" instead of "opt-in > 0".